src/applications/disability-benefits/all-claims/pages/unemployabilityFormUpload.js

```diff
@@ -0,0 +1,61 @@
+import fileUploadUI from 'us-forms-system/lib/js/definitions/file';
+import environment from '../../../../platform/utilities/environment';
+import { FIFTY_MB } from '../constants';
+import { DocumentDescription } from '../content/uploadFormDocuments';
+
+export const uiSchema = {
+  'ui:title': 'Upload VA Form 21-8940',
+  'ui:description': DocumentDescription,
+  form8940Upload: fileUploadUI('', {
+    itemDescription: 'Document Type (VA Form 21-8940)',
```

Comment:
Hmm my context to this code is starting to slip with time! I'm not seeing the title show up anywhere when added to the first argument? I believe I put it there so it would show up on the upload tab(see screenshot above)...

**With `ui:title` added** (not seeing it render anywhere?)
![image](https://user-images.githubusercontent.com/22040793/48856595-d0d6bc80-ed84-11e8-9339-7ca23547b868.png)

Reply:
You also have / had a `hideLabelText: true` which I'm pretty sure just hides whatever you put in as that first arg :)